### DISCUSSION TITLE
Handholding for newbs

### DISCUSSION THREAD

**Discussion by dscotese** [Ride-The-Lightning/RTL#822] [General]
I think the best summary for this "feature request" is a description of the "build process" or something like that in the readme.  I saw a folder in this repo with "Angular pre-build code" in a comment, which suggests to me that there is some work to do when building.  I'm new to Angular and TypeScript, but not to coding, github, javascript, Bitcoin, Lightning, or nodeJS.

I cloned RTL to play with it. I added an entry to the node array in RTL-Config.json and set lnImplementation to "KG". I changed navMenu.ts to add a new element to MenuRootNode, KGChildren, which has only one child named "Dashboardx".  I also changed side-navigation.component.ts so that when a configuration has lnImplementation set to KG, filterSideMenuNodes() will load KGChildren.  The side menu still has "Dashboard", not "Dashboardx".  I edited navMenu.ts again to make it "DashboardL" for LNDChildren, but it still says "Dashboard."  I guess I'm missing a step somewhere (Angular prebuild?).  I have crypto to pay for help, or maybe, if I understand it well enough, I can start contributing to RTL too.

**Comment by ShahanaFarooqui** [comment]
@dscotese thanks for showing the interest in contributing to RTL. Please check ‘ https://github.com/Ride-The-Lightning/RTL/blob/master/docs/Contributing.md’ link to better understand the coding steps. 

I am also contemplating about making some videos to explain the project and coding process. Would that be more helpful?

Also please note that this project is Bitcoin only and we do not accept any PR implemented on shitcoins.

**Comment by saubyk** [comment]
Since this is not an issue directly related to any feature or bug, I am moving it to discussions.
